apps/ledger-live-mobile/src/components/RootNavigator/types/AccountsNavigator.ts

```diff
@@ -34,4 +34,15 @@ export type AccountsNavigatorParamList = {
   [ScreenName.Asset]: {
     currency: CryptoOrTokenCurrency;
   };
+  [ScreenName.AccountsList]: {
+    sourceScreenName: ScreenName;
+    showHeader?: boolean;
+    canAddAccount?: boolean;
+    isSyncEnabled?: boolean;
+  };
+  [ScreenName.AssetsList]: {
```

Comment:
i don't think you need this Screen in Accounts Navigator

Reply:
Removed ✅